modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -543,8 +501,23 @@ namespace gapi {
     /** @} */
 
     GAPI_EXPORTS GKernelPackage combine(const GKernelPackage  &lhs,
-                                        const GKernelPackage  &rhs,
-                                        const cv::unite_policy policy);
+                                        const GKernelPackage  &rhs);
+    /**
+     * @brief cv::use_only() is a special combinator which hints G-API to use only
+     * kernels specified in cv::GComputation::compile() (and not to extend kernels available by
+     * default with that package).
+     */
+
+    class GAPI_EXPORTS use_only
+    {
+        public:
+        use_only(const GKernelPackage& pkg) : _pkg(pkg)            {};
+        use_only(GKernelPackage&& pkg)      : _pkg(std::move(pkg)) {};
+        GKernelPackage getPackage() const { return _pkg; }
```

Comment:
What is the reason in this function if `_pkg` is anyway public?

I'd make it struct and drop this method at all, I think

Reply:
Fix